keccak-hash/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "keccak-hash"
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
and this (until we resolve #146)

Reply:
I will obey, but are you sure that features available in `keccak-hash` used the `libc` feature from `primitive-types` in the past?

Reply:
it's not about features anymore, once you expose a type in a lib
https://github.com/paritytech/parity-common/blob/bf5879459ade112d128312408d1f190f3839a93b/keccak-hash/src/lib.rs#L35

and upgrade a dependency with that type in a semver breaking way, the breakage speads like a virus into the lib